updating/updating-restricted-network-cluster.adoc

```diff
@@ -0,0 +1,50 @@
+[id="updating-restricted-network-cluster"]
+= Updating a restricted network cluster
+include::modules/common-attributes.adoc[]
+:context: updating-restricted-network-cluster
+
+toc::[]
+
+You can upgrade a restricted network {product-title} cluster by using the `oc` command-line interface (CLI).
+
+A restricted network environment is the one in which your cluster nodes cannot access the internet. For this reason you must mirror the images to a file system that disconnected from that environment and then bring that host or removable media across that gap.
```

Comment:
"Mirror the image to a file system, then bring that host or removable media across that gap" is only one case, to be more accurate, we also need to take "directly push the image to the local container registry" into consideration (refer to "If the local container registry and the cluster are connected to the mirror host, directly push the release images to the local registry and apply the ConfigMap to the cluster by using following command" section).